### ISSUE TITLE
Implemented in a genAI add-on for Blender

### ISSUE THREAD

**Issue by tin2tin** [ChenyangSi/FreeU#14]
Thank you for sharing your work! 

Using this wrapper: https://github.com/lyn-rgb/FreeU_Diffusers I was able to implement it as an experimental feature in the genAI add-on for Blender: https://github.com/tin2tin/Pallaidium

Here are some early tests: 
![image](https://github.com/ChenyangSi/FreeU/assets/1322593/2155fb6e-8566-4749-97ca-355fc9f654b6)
![image](https://github.com/ChenyangSi/FreeU/assets/1322593/482ebd30-b2ee-4c6b-b8c9-2ba2d2a58ae8)

These are rendered with SDXL - maybe they're a bit too dark?

**Comment by ChenyangSi** [comment]
you can try to reduce b

**Comment by tin2tin** [comment]
Using the values from here: https://wandb.ai/nasirk24/UNET-FreeU-SDXL/reports/FreeU-SDXL-Optimal-Parameters--Vmlldzo1NDg4NTUw?accessToken=6745kr9rjd6e9yjevkr9bpd2lm6dpn6j00428gz5l60jrhl3gj4gubrz4aepupda

Mind-blowing!!!
![image](https://github.com/ChenyangSi/FreeU/assets/1322593/cac9512c-1c79-4fd6-ad6c-e9528232616d)

**Comment by ChenyangSi** [comment]
Can I put your results in readme? https://github.com/ChenyangSi/FreeU#results-from-the-community

**Comment by tin2tin** [comment]
Of course, you can! And thank you for sharing this magic bullet of yours!
